src/graphql-server/holochainClient.js

```diff
@@ -1,14 +1,17 @@
 import { connect as hcWebClientConnect } from '@holochain/hc-web-client'
 import { get } from 'lodash/fp'
-import { HYLO_INSTANCE_ID, HAPP_STORE_INSTACE_ID } from 'config/dnaconfig'
+import { HYLO_INSTANCE_ID, HAPP_STORE_INSTANCE_ID } from 'config/dnaconfig'
+import mockCallZome from 'mock-dnas/mockCallZome'
+
+const MOCK_DNA_CONNECTION = true
 
 export const HOLOCHAIN_LOGGING = true
-const holochainClients = {}
+let holochainClient
 
-async function initAndGetHolochainClient (instanceId) {
-  if (holochainClients[instanceId]) return holochainClients[instanceId]
+async function initAndGetHolochainClient () {
+  if (holochainClient) return holochainClient
```

Comment:
y u no braces? :(

Reply:
you don't like those one-liner ifs?

Reply:
Not really. I realise this is probably nitpicking, but on the other hand, there's also [dangling elses](https://en.wikipedia.org/wiki/Dangling_else)

Reply:
I don't think I've ever run into a situation where I would want to nest one-liner ifs, I would always break it into multiple lines in that case. The one liner is always (or almost always) in the case where I'm conditionally returning from the function.